modules/diskq/qdisk.c

```diff
@@ -50,6 +51,10 @@
 
 #define QDISK_HDR_VERSION_CURRENT 3
 
+#define FLOCK_FILENAME "disk-buffer.flock"
```

Comment:
Can we add `syslog-ng` to the name?
The disk-buffer dir should be private and dedicated for syslog-ng, but sometimes it's just `/var`.

For example:
`syslog-ng-disk-buffer.flock`
`syslog-ng-disk-buffer.dirlock`
`syslog-ng-disk-buf-dir.lock`
`syslog-ng.diskqdirlock`

Reply:
Yes, added it :)